api/v1_users_authorized_apps.go

```diff
@@ -0,0 +1,77 @@
+package api
+
+import (
+	"time"
+
+	"bridgerton.audius.co/api/dbv1"
+	"bridgerton.audius.co/trashid"
+	"github.com/gofiber/fiber/v2"
+)
+
+type GetUsersAuthorizedAppsQueryParams struct {
+	Limit  int `query:"limit" default:"50" validate:"min=1,max=100"`
```

Comment:
50 by default eh? Out of curiosity, does that match the values in the swagger spec? (if they are even defined)

Reply:
okay 10